ruby/ql/lib/codeql/ruby/security/ServerSideRequestForgeryQuery.qll

```diff
@@ -0,0 +1,33 @@
+/**
+ * Provides a taint-tracking configuration for detecting
+ * "Server side request forgery" vulnerabilities.
+ *
+ * Note, for performance reasons: only import this file if `Configuration` is needed,
+ * otherwise `ServerSideRequestForgeryCustomizations` should be imported instead.
+ */
+
+import codeql.ruby.DataFlow::DataFlow::PathGraph
```

Comment:
Is this missing `import ruby` as the first statement?
AFAIK `import ruby` should be the first statement for [caching](https://github.com/github/codeql/commit/deb6a6e5c69ee7ea54b97561554ed4c0cee471fe) (?) reasons.

Reply:
Ah interesting! I didn't know that, thanks for pointing it out!

Reply:
Actually, I wonder if that only applies to query files. The corresponding `ServerSideRequestForgery.ql` file does `import ruby`, so I think we may be OK.

Reply:
I thought I've read this somewhere, but I'm not sure anymore.
If you should find a definite answer, please share it with me :)

Reply:
@hvitved ^